docs/source/results.md

```diff
@@ -20,7 +20,8 @@ For example, for a `project_name` of `ygradient_cylinder`, the intermediate imag
 
 ## Output Meshes
```

Comment:
This comment is for line 18. Since you mention that there are four images, you should add the filename for the fourth intermediate image.

Reply:
There are only 3 images produced. I updated the text.